extensions/example-extension/page.tsx

```diff
@@ -1,12 +1,19 @@
-import { LensRendererExtension, Component } from "@k8slens/extensions";
-import { CoffeeDoodle } from "react-open-doodles";
-import path from "path";
+import { Component, LensRendererExtension, Navigation } from "@k8slens/extensions";
 import React from "react";
+import path from "path";
+import { observer } from "mobx-react";
+import { CoffeeDoodle } from "react-open-doodles";
+
+export const exampleId = Navigation.createPageParam({
+  name: "exampleId",
+  defaultValue: "demo",
+});
```

Comment:
I think we need a strong page-to-param association to guard against wires getting crossed as people use different params for different pages. Also, I think the name is counter-intuitive because it sounds like we're declaring parameters, but we're actually getting a way to access them:
```javascript
Navigation.getPageParam({
  pageId: 'example',
  name: 'exampleId',
  defaultValue: 'demo',
});
```
And then in `renderer.tsx`, you declare the params the page will use by just adding the `params` object to the page declaration, like:
```javascript
  globalPages = [
    {
      id: "example",
      components: {
        Page: () => <ExamplePage extension={this}/>,
      },
      params: {
        exampleId,
      }
    }
  ];
```

Reply:
Probably it was just a wrong example for this usage (or need some renaming).